util/venv-dependencies.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -e
+
+# quiet=--quiet
+quiet=
+
+#### Options end here.
+
+willexit=
+if [ "$#" -ne 1 ]; then
+   echo "
+Usage: $0 new-directory
+
+This script installs Streisand builder dependencies. It depends on
+a working Python 2.7 'pip' command on your PATH. I'll install 
```

Comment:
I think since the other Streisand scripts favour a non-first-person style of prose we should avoid "I'll" & "I" in prompts/messages.

Reply:
Got it.